File: Domain/Termini.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace Domain
{
    public class Termini
    {
        public int Id {get;set;}
        public string Koha {get;set;}
        public DateTime Data {get;set;}
        public string PacientiId {get;set;}
        public Pacienti Pacienti {get;set;}
        public ICollection <Kontrolla> Kontrollat { get; set; } = new List<Kontrolla>();
    }
}

File: client-app/src/app/models/termini.ts
import { Doktori } from "./doktori";
import { Kontrolla } from "./kontrolla";
import { Pacienti } from "./pacienti";

export interface Termini {
  id?: string;
  koha: string;
  data: Date | null;
  pacientiId?: string;
  kontrolla?: Kontrolla[];
}